content/dstu2/binary.md

```diff
@@ -36,4 +36,49 @@ Retrieving the Binary resource with its native mime type
 <%= headers 200, 'Content-Type': 'text/html' %>
 <%= html(:dstu2_binary_html) %>
 
+## Operation autogen-ccd-if
```

Comment:
Updated to `Operation: autogen-ccd-if`
https://github.com/cerner/fhir.cerner.com/pull/104/commits/2c670a589109284290cc69219df754530d482876